packages/redux/src/users/personalIds/selectors.ts

```diff
@@ -0,0 +1,82 @@
+import {
+  getDefaultPersonalId as getDefaultPersonalIdFromReducer,
+  getError as getErrorFromReducer,
+  getIsLoading as getIsLoadingFromReducer,
+  getResult as getResultFromReducer,
+} from './reducer';
+import { getPersonalIds as getPersonalIdsFromReducer } from '../reducer';
+import type { StoreState } from '../../types/storeState.types';
+import type { UserPersonalIdsState } from './types';
+import type { UsersState } from '../types/state.types';
+
+/**
+ * Returns the loading status of the personal ids area.
+ *
+ * @param state - Application state.
+ *
+ * @returns Loader status.
+ */
+export const areUserPersonalIdsLoading = (
+  state: StoreState,
+): UserPersonalIdsState['isLoading'] =>
```

Comment:
Suggestion: If the selector is simple like this one, you can remove the return type annotation as it should be inferred by the body expression. Review this selector and other selectors where that rule applies

Reply:
done :)